README.md

```diff
@@ -7,12 +7,13 @@ Make [Lula](https://github.com/defenseunicorns/lula) available to your GitHub Ac
 ### Linting OSCAL file against schema
 ```yaml
 - name: Setup Lula
-  uses: defenseunicorns/lula/setup@v1
+  uses: defenseunicorns/lula-action/setup@v1
```

Comment:
can you change the version to main on all of these real quick? Until we sort out which version we are going to use.